modules/oe_editorial_corporate_workflow/config/install/workflows.workflow.corporate_workflow.yml

```diff
@@ -0,0 +1,149 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - content_moderation
+id: corporate_workflow
+label: 'Corporate Workflow'
+type: content_moderation
+type_settings:
+  states:
+    archived:
+      published: false
+      default_revision: true
+      label: Archived
+      weight: 1
+    draft:
+      published: false
+      default_revision: false
+      label: Draft
+      weight: -4
+    expired:
+      published: false
+      default_revision: true
+      label: Expired
+      weight: 2
+    needs_review:
+      published: false
+      default_revision: false
+      label: 'Needs Review'
+      weight: -3
+    published:
+      label: Published
+      published: true
+      default_revision: true
+      weight: 0
+    request_validation:
+      published: false
+      default_revision: false
+      label: 'Request Validation'
+      weight: -2
+    unpublished:
```

Comment:
Yes, I know, but I definitely foresee the need for this. Since the workflow design is based on D7 experiences, where the published and unpublished state were node states rather than workbench states I added this. We can confirm this with business.

Reply:
Meanwhile I remove it from this delivery.